packages/ember-metal/tests/utils_test.js

```diff
@@ -10,6 +10,7 @@ QUnit.module('Ember Metal Utils');
 QUnit.test('inspect outputs the toString() representation of Symbols', function() {
   // Symbol is not defined on pre-ES2015 runtimes, so this let's us safely test
   // for it's existence (where a simple `if (Symbol)` would ReferenceError)
+  /* jshint latedef: false */
   let Symbol = Symbol || null;
```

Comment:
Submitted https://github.com/emberjs/ember.js/pull/14531 to fix this specific test in release/lts-2-8 branches, and they were already fixed in https://github.com/emberjs/ember.js/pull/14267 (see [here](https://github.com/emberjs/ember.js/blob/master/packages/ember-utils/tests/inspect_test.js#L5)).